Scuti/Scripts/UI/Store/Offer Details/OfferInfoPresenter.cs

```diff
@@ -45,6 +45,9 @@ public class Model : Presenter.Model {
         [SerializeField] GameObject scutiPromo;
 
         protected override void OnSetState() {
+
+            string text = Data.Title;
+            text.Replace("&amp;", "&");
```

Comment:
@Jpablogs8 do we still need this since we are using HTMLDecode?

Reply:
I'll remove it

Reply:
@Jpablogs8